apps/measures/templates/apps_measures/includes/measures_list_item.html

```diff
@@ -19,8 +19,7 @@ <h3 class="title">{{ content.page_title|truncatechars:102 }}</h3>
             <p class="text">{{ content.contact_organisation_name }}</p>
             <p class="u-b-none">
                 {% for foa in content.get_fields_of_action %}
-                    {{ foa.subset_name }}
-                    {% if not forloop.last %}, {% endif %}
+                    {{ foa.subset_name }}{% if not forloop.last %}, {% endif %}
```

Comment:
the issue says 
> expected behaviour: Each field of action should be in a separate line
however this just fixes a whitespace in the comma separated list. Is this intentional?

Reply:
ahh didn't read the issue, thought it was same as before, sorry